ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.view.js

```diff
@@ -10,14 +10,35 @@
  *
  **/
 
-const { reactToMarionette } = require('component/transmute')
-const LocationView = reactToMarionette(require('react-component/location'))
+const React = require('react')
 
-if (process.env.NODE_ENV !== 'production') {
-  module.hot.accept('react-component/location', () => {
-    LocationView.reload(require('react-component/location'))
-  })
-}
+const withAdapter = Component =>
```

Comment:
❓ Is it possible to make this it's own module?

Reply:
It was its own module, `transmute` which I'm removing because we should be using the template method instead. When I get the opportunity, I'll probably collapse all the `withAdapter` calls as well but I didn't want to make too many changes.